src/tax/display-settings.md

```diff
@@ -13,7 +13,15 @@ Showing prices that include and exclude tax can be confusing to the customer. To
 ![]({% link images/images/config-sales-tax-price-display-settings.png %}){: .zoom}
 [_Price Display Settings_]({% link configuration/sales/tax.md %})
 
-## Step 1: Configure price display settings
+## Configure price display settings
+
+When the calculation of taxes, rates, and classes is finished, taxes will be calculated according to the specified settings. However, the display of taxes in the catalog, shopping cart, orders, invoices, and credit memos should be set up, as part of the customer experience.
+
+It is best practice to display prices including taxes, or both, including taxes and excluding taxes so that customers know how much they will change before placing an order.
```

Comment:
```suggestion
It is a best practice to display prices with the associated taxes (either including taxes, or both the including taxes and excluding taxes) so that customers know how these calculations are applied before placing an order.
```